src/test/java/WordsTest.java

```diff
@@ -0,0 +1,31 @@
+import domain.Word;
+import domain.Words;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class WordsTest {
+
+    @Test
+    void getArrayLocation(){
+        //given
+        Words words = Words.of(Arrays.asList("aaaaa", "bbbbb"));
+
+        //when
+        Word answers = words.answer(LocalDate.of(2021, 6, 20));
+
+        //then
+        Assertions.assertEquals(answers, new Word("bbbbb"));
+    }
+
+    @Test
+    void validation() {
+        Words words = Words.of(Arrays.asList("aaaaa"));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> {
+            words.matchingAnswer(new Word("bbbbb"));
+        });
+    }
```

Comment:
여기도 다른 테스트들과 일관되게 테스트에 대한 설명을 `@DisplayName`으로 달아줘도 좋을 것 같습니다 :)
추가로 테스트 계층구조에 대해서도 참고해보시면 좋을 것 같네요 😊
- https://johngrib.github.io/wiki/junit5-nested/

```suggestion
    @Test
    @DisplayName("....")
    void getArrayLocation(){
        //given
        Words words = Words.of(Arrays.asList("aaaaa", "bbbbb"));

        //when
        Word answers = words.answer(LocalDate.of(2021, 6, 20));

        //then
        Assertions.assertEquals(answers, new Word("bbbbb"));
    }

    @Test
    @DisplayName("....")
    void validation() {
        Words words = Words.of(Arrays.asList("aaaaa"));
        Assertions.assertThrows(IllegalArgumentException.class, () -> {
            words.matchingAnswer(new Word("bbbbb"));
        });
    }
```